lib/embed/embed.dart

```diff
@@ -278,7 +278,9 @@ class PlaygroundMobile {
   }
 
   void _export() {
-
+    window.open(
+        "/newpad?dart=${context.dartSource}&html=${context.htmlSource}&${context.cssSource}",
```

Comment:
You need a `&css=` here, and we'll need to url encode the source. This may work: `Uri.encodeQueryComponent`.